### ISSUE TITLE
The value of the generated audio's silent segment is not 0?

### ISSUE THREAD

**Issue by hwong39** [fatchord/WaveRNN#22]

**Comment by fatchord** [comment]
@hwong39 I would imagine most training data will have some amount of floor noise in the recordings. Perhaps this would explain it?

**Comment by geneing** [comment]
Note that audio is generated by sampling from a random distribution, with network output giving probabilities for each value of amplitude. Here's a snippet from generate function:

>                posterior = F.softmax(x, dim=1).view(-1)
>                distrib = torch.distributions.Categorical(posterior)
>                sample = 2 * distrib.sample().float() / (self.n_classes - 1.) - 1.
>                output.append(sample)
